RecoTauTag/RecoTau/interface/RecoTauQualityCuts.h

```diff
@@ -91,6 +92,9 @@ namespace reco {
         return output;
       }
 
+      /// Declare all parameters read from python config file
+      static void fillDescriptions(edm::ParameterSetDescription& descriptions);
```

Comment:
I'd suggest to rename this function to
```suggestion
      static void fillPSetDescription(edm::ParameterSetDescription& descriptions);
```
to follow a convention used elsewhere (including the [helper plugins](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#A_Plugin_Module_Using_Another_He)). The suggested name also signifies that the function fills a description for one PSet instead of arbitrary number of modules.